modules/persistence/src/commonMain/sqldelight/entities/Account.sq

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE account (

```

Comment:
Is table creation sqldelight responsibility?

Reply:
I followed the "Fresh Schema" guide from here https://cashapp.github.io/sqldelight/2.0.2/jvm_postgresql/

Reply:
Not sure if you can combine them both, but we should follow the migration approach, ensuring a consistent DB over time